Client/src/services/saveLastCityToLS.js

```diff
@@ -0,0 +1,26 @@
+const saveLastCityToLS = (city, id) => {
+  let stuffInLs = JSON.parse(localStorage.getItem('lastCity'));
+
+  function checkIfCityExists() {
+    let exists = Boolean;
+    stuffInLs
```

Comment:
Here is your code:
`const checkIfCityExists = () => savedCities.some(savedCity => savedCity.city === city);`

Reply:
thanks, didn't think of it